src/crewai/llm.py

```diff
@@ -59,6 +59,7 @@
 
 load_dotenv()
 
+litellm.suppress_debug_info = True
```

Comment:
This will prevent logs like "give feedback / get help" in lower_s from being printed